src/clj/witan/send/goal_seek.clj

```diff
@@ -55,18 +67,61 @@
 (defn target-pop-exceeded? [pop target-pop]
   (> pop (apply max target-pop)))
 
-(defn target-results [m start end step base-config target]
-  "Takes a map of keys partially matching a transition, a start, end and step range to modify
-   the transition by, a template config and a map containing a target population range and year
+(defn assoc-transition-params [config m]
+  (let [state (create-transition-modifier-seq m 1 2 1)]
```

Comment:
worth putting in an example of what the before an dafter states of the config are here?